release-notes/index.dita

```diff
@@ -174,6 +215,20 @@
 
       <section id="enhancements">
         <title>Enhancements and changes</title>
+        <div outputclass="div-index">
```

Comment:
What's the `@outputclass` for?

Reply:
The `<div>` is new and is used to group index entries below a `<section>` title. Doing so, however, while solving one problem creates another -- namely, an extra space since the `<div>` is since as a block. The `@outputclass` is present so that special handling can be done in the future, if desired.